frontend/src/lib/components/expander/Expander.svelte

```diff
@@ -136,7 +136,8 @@
 
   /* This is needed to add padding to collapse content only when collapse is open */
   .collapse:not(.collapse-close) > input[type='checkbox']:checked ~ .collapse-content {
-    padding: 0.7rem;
+    padding-right: calc(env(safe-area-inset-right, 0px) + 1.25rem);
+    padding-left: calc(env(safe-area-inset-left, 0px) + 1.25rem);
```

Comment:
Could you add a comment that explain why the `safe-area-inset-right` is needed? Is it intentional that there is no longer any padding on the vertical axis?

Reply:
The component looks better without padding on the vertical axis.